editor/create_dialog.cpp

```diff
@@ -289,14 +289,12 @@ void CreateDialog::_configure_search_option_item(TreeItem *r_item, const String
 
 	bool can_instantiate = (p_type_category == TypeCategory::CPP_TYPE && ClassDB::can_instantiate(p_type)) ||
 			p_type_category == TypeCategory::OTHER_TYPE;
-	bool is_virtual = ClassDB::class_exists(p_type) && ClassDB::is_virtual(p_type);
+	bool instantiable = !(ClassDB::class_exists(p_type) && ClassDB::is_virtual(p_type)) && can_instantiate;
```

Comment:
```suggestion
	bool instantiable = can_instantiate && !(ClassDB::class_exists(p_type) && ClassDB::is_virtual(p_type));
```

We should evaluate `can_instantiate` first, because, well, it's already evaluated.